### PR TITLE
Configure Stale Bot

### PR DESCRIPTION
**Pull request by edalex-ian**
Following the Community Developer Meeting it was agreed to use Stale Bot
to try and keep our Issue List under control. After enabling it on our
organisation, I'm now following the instructions as provided in the
Installed GitHub Apps for Stale.

These are essentially the recommended defaults.

As per https://github.com/organizations/equella/settings/installations/1802878

### CODE REVIEW

File: .gitlab/stale.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
```
Comment (ChristianMurphy) [inline .gitlab/stale.yml]: the docco for stale bot mentions `.github/stale` as the folder.
https://github.com/probot/stale#usage

will `.gitlab/stale` also work?

File: .gitlab/stale.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
```
Comment (ChristianMurphy) [inline .gitlab/stale.yml]: some other labels that might be exempt-able.
- tech debt
- roadmap
- edelex
- unicon se

thoughts? 💭

File: .gitlab/stale.yml (hunk 1 above)
Comment (edalex-ian) [reply .gitlab/stale.yml]: Ah bugga. Sorry, my bad. Jumping between too many different systems. Will fix.

File: .gitlab/stale.yml (hunk 2 above)
Comment (edalex-ian) [reply .gitlab/stale.yml]: Good idea. I'll add them. 👍

**Pull request merged**